.github/workflows/dist.yml

```diff
@@ -42,49 +42,6 @@ jobs:
           name: ${{ matrix.os }}-wheel
           path: ./wheelhouse/*.whl
           if-no-files-found: error
-  test_non_linux_wheels:
```

Comment:
These jobs were brittle and are redundant - they're testing that the stable api works